the-blue-alliance-ios/View Controllers/MyTBA/MyTBATableViewController.swift

```diff
@@ -146,6 +146,7 @@ class MyTBATableViewController<T: MyTBAEntity & MyTBAManaged, J: MyTBAModel>: TB
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let fallbackCell = UITableViewCell()
         fallbackCell.textLabel?.text = "----"
+        fallbackCell.selectionStyle = .none
```

Comment:
This is related to https://github.com/the-blue-alliance/the-blue-alliance-ios/issues/177#issuecomment-413750734